common/src/test/java/org/keycloak/common/util/HtmlUtilsTest.java

```diff
@@ -0,0 +1,25 @@
+package org.keycloak.common.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.keycloak.common.util.HtmlUtils;
+
+public class HtmlUtilsTest {
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+  @Rule
+  public Timeout globalTimeout = new Timeout(10000);
```

Comment:
Both these rules and their associated imports can be removed from this test class